profiles/arch/amd64-fbsd/clang/make.defaults

```diff
@@ -6,3 +6,6 @@ CFLAGS="-O2 -pipe"
 CXXFLAGS="-stdlib=libc++ ${CFLAGS}"
 FFLAGS="${CFLAGS}"
 FCFLAGS="${CFLAGS}"
+
+# sys-devel/clang require static-analyzer, bug 539858
+BOOTSTRAP_USE="${BOOTSTRAP_USE} static-analyzer"
```

Comment:
I used it because it was a problem of stage1.

sys-devel/llvm is enabled static-analyzer by default.
However, it is disabled by USE="-*" when making stage1.

Please comment in bugzilla. Might get a more detailed comments by aballier.